commands/init/index.js

```diff
@@ -50,46 +51,65 @@ class InitCommand extends Command {
     });
   }
 
+  get hasExistingLernaConfig() {
+    return !!this.project.version;
+  }
+
   ensurePackageJSON() {
     let chain = Promise.resolve();
 
     if (!this.project.manifest) {
       this.logger.info("", "Creating package.json");
 
       // initialize with default indentation so write-pkg doesn't screw it up with tabs
-      chain = chain.then(() =>
-        writeJsonFile(
-          path.join(this.project.rootPath, "package.json"),
-          {
-            name: "root",
-            private: true,
-          },
-          { indent: 2 }
-        )
-      );
+      chain = chain.then(() => {
+        const pkg = {
+          name: "root",
+          private: true,
+        };
+
+        if (!this.hasExistingLernaConfig) {
```

Comment:
The more I look at this logic, the more I realize that the `ensurePackageJSON` and `ensureLernaConfig` functions should be combined. Both config files are dependent on each other, such as for workspaces support.